backend.py

```diff
@@ -10,20 +10,22 @@
 
 MAX_DAMAGE_VALUE = 10
 
+
 class Robot:
     def __init__(self, direction, coordinates, name):
         self.direction = direction
         self.coordinates = coordinates
         self.start_coordinates = coordinates
-        # program = cards on hand, list.
-        # currently testing's value, to be removed
-        self.program = [RotationCard(200, Rotation.LEFT), MovementCard(100, 2)]
+        self.program = []
         self.lives = 3
         self.flags = 0
         self.damages = 4
         self.power_down = False
         self.name = name
 
+        # special testing attribute, start tile number
+        self.stn = 0
```

Comment:
Tohle by bylo lepší pojmenovat `_start_tile_number`, nebo dokonce `_tile_number_for_tests`:
* Ze `stn` není poznat, co to vlastně je (a jestli je to důležité).
* Podtržítko na začátku je standardní způsob, jak říct že to ostatní nemají používat.

Reply:
Teď tady zkouším jinou věc: ja tento atribut asi nemusím vůbec inicializovat u robota, můžu ho nastavit a použivat pouze v testech. Testy prochází, i když žádný takový atribut přímo v třídě `Robot` není, což buď znamená, že můžu to zde vypoustit, nebo že mám v kódu nějaký nedodělek (viz stávající stav: přepsala jsem atribut v backendu, ale soubor test_effects obsahuje ještě `robot.stn` - bez škody pro test).